src/main/kotlin/com/cultureamp/eventsourcing/RelationalDatabaseEventStore.kt

```diff
@@ -81,22 +111,27 @@ class RelationalDatabaseEventStore<M : EventMetadata> @PublishedApi internal con
                         val metadata = objectMapper.writeValueAsString(event.metadata)
                         validateSerialization(domainEventClass, body, metadata)
                         val eventType = eventTypeResolver.serialize(domainEventClass)
-                        val insertResult = events.insert { row ->
-                            row[events.aggregateSequence] = event.aggregateSequence
-                            row[events.eventId] = event.id
-                            row[events.aggregateId] = aggregateId
-                            row[events.aggregateType] = event.aggregateType
-                            row[events.eventType] = eventType
-                            row[events.createdAt] = event.createdAt
-                            row[events.body] = body
-                            row[events.metadata] = metadata
+                        val insertResult = eventsSinkTable.insert { row ->
+                            row[eventsSinkTable.aggregateSequence] = event.aggregateSequence
+                            row[eventsSinkTable.eventId] = event.id
+                            row[eventsSinkTable.aggregateId] = aggregateId
+                            row[eventsSinkTable.aggregateType] = event.aggregateType
+                            row[eventsSinkTable.eventType] = eventType
+                            row[eventsSinkTable.createdAt] = event.createdAt
+                            row[eventsSinkTable.body] = body
+                            row[eventsSinkTable.metadata] = metadata
                         }
-                        val insertedSequence = insertResult[events.sequence]
-                        eventsSequenceStats.replace {
-                            it[eventsSequenceStats.eventType] = eventType
-                            it[eventsSequenceStats.sequence] = insertedSequence
+                        val dryRunMode = (events != eventsSinkTable)
+                        if (dryRunMode) {
+                            SequencedEvent(event, -1)
```

Comment:
Why doesn't need the sequence?

Reply:
Every dry-run table should be brand new. We can add a validator to check if it is not a new table. It refuses to do dry-run.

Reply:
> Why doesn't need the sequence?

Because it's dry-run, it's not actually going to raise any events that projectors can consume. 
The only current use-case for the sequence is for a command to wait until specified projectors have processed the events produced by the command. In this case there will be no events the projectors can consume, so there should be no waiting

Reply:
> Every dry-run table should be brand new. We can add a validator to check if it is not a new table. It refuses to do dry-run.

You mean at deploy time? I don't think that's true even for the first time dry-run code is deployed - doesn't it create 2 instances of the process? only the first would create the table